nn_text_classifier.py

```diff
@@ -219,6 +219,8 @@ def import_csv(filename):
         predict_this.append(format_sentence(phrase))
         sentences.append(phrase)
     answers = []
+    print(predict_this)
+    exit()
```

Comment:
Why does it exit here? Do we need this file at all anymore? It looks like it was moved into the sentiment folder.

Reply:
I actually don't see this on my local nn_review_classifier. But I do have another file called nn_review_classifier.py.swp?